app/generators/radio_generator.rb

```diff
@@ -67,11 +72,18 @@ def radio_otherwise(question)
   end
 
   def radio_otherwise_option(question)
-    tag.input(name: answer_name(idify(question[:id])),
-              type: 'radio',
-              id: idify(question[:id], question[:raw][:otherwise_label]),
-              value: question[:raw][:otherwise_label],
-              required: true,
-              class: 'otherwise-option')
+    value = question[:raw][:otherwise_label]
+    tag.input(
+      **decorate_with_previous_value(
+        question,
+        question[:id],
+        { name: answer_name(idify(question[:id])),
+          type: 'radio',
+          id: idify(question[:id], question[:raw][:otherwise_label]),
+          value: value,
+          required: true,
+          class: 'otherwise-option' }
+      ) { |previous_value| [:checked, previous_value == value] }
+    )
   end
 end
```

Comment:
Why not add support for the checkbox question type also? I understand leaving out the support for sliders, because sliders are quickly and easy to set by the user, but a checkbox seems to me no different than a dropdown or radio question?